packages/components/CONTRIBUTING.md

```diff
@@ -180,6 +183,65 @@ All new component should be styled using [Emotion](https://emotion.sh/docs/intro
 
 Note: Instead of using Emotion's standard `cx` function, the custom [`useCx` hook](/packages/components/src/utils/hooks/use-cx.ts) should be used instead.
 
+
+### Deprecating styles
```

Comment:
Should this deprecation strategy be recommended also beyond styles? It seems like a reasonable strategy to use also for behavioural changes, API changes, etc...

Reply:
I do see this as a special case of deprecating APIs, where there is no actual prop to deprecate and it's just a change in default behavior. Looking at the instances in `components/*` where `deprecated()` is called, I'm not sure this kind of "change in default" deprecation has ever happened. Are you aware of any?

I did choose the `__next*` prefix so it would be appropriate for behavioral changes as well though, so it probably would be generally applicable when it happens. I think we could reuse or refactor this section when we write a more detailed deprecation guide for components.

By the way I just learned about [this section](https://github.com/WordPress/gutenberg/blob/docs/deprecating-styles/docs/contributors/code/backward-compatibility.md#dev-notes) that asks us to add a "Needs Dev Note" label to the PR. I'll add this into the doc.

Reply:
> Are you aware of any?

On top of my mind, no. Hey @sarayourfriend (sorry for the ping!) — do you ever remember an instance of a deprecation where the meaning of a style/prop was changed (causing a potentially breaking change)?

> I did choose the __next* prefix so it would be appropriate for behavioral changes as well though, so it probably would be generally applicable when it happens. I think we could reuse or refactor this section when we write a more detailed [deprecation guide for components](https://github.com/WordPress/gutenberg/blob/docs/deprecating-styles/docs/contributors/code/backward-compatibility.md#how-to-preserve-backward-compatibility-for-a-react-component).

Sounds good to me. Having a defined way to "safely" introduce a breaking change is great and we should absolutely iterate on this procedure in order to be able to apply it to more deprecation scenarios

> By the way I just learned about [this section](https://github.com/WordPress/gutenberg/blob/docs/deprecating-styles/docs/contributors/code/backward-compatibility.md#dev-notes) that asks us to add a "Needs Dev Note" label to the PR. I'll add this into the doc.

TIL !

Reply:
Yeah, that's what I thought as well.

> Although, does this count? https://github.com/WordPress/gutenberg/pull/33490

Not sure if it does, I'd probably see it as more of a bug fix — `isScrollable` was implicitly introduced to `CardBody` in a previous PR, and the PR that you highlighted changes its default value to _fix_ that regression.

Reply:
Right right... I feel like I remember seeing a bug raised where we'd accidentally left a deprecation or prop-meaning change out of the changelogs somewhere and it'd caused problems for someone consuming the package. Just can't remember how to find it now 😞

Reply:
I believe the original "breaking change" was caused by my refactor of `Card` in #32566